.bldr.toml

```diff
@@ -458,6 +458,7 @@ paths = [
   "lib/reporting/*",
   "lib/secrets/*",
   "lib/stringutils/*",
+  "lib/sshutils/*",
```

Comment:
This should come above stringutils to pass the build

Reply:
done